### ISSUE TITLE
MeshCentral crashed

### ISSUE THREAD

**Issue by hunter-nl** [Ylianst/MeshCentral#280]
I just got this error:
```
-------- 6/7/2019, 10:58:33 AM ---- 0.3.6-c --------

/opt/meshcentral/node_modules/mongodb/lib/utils.js:132
      throw err;
      ^

ReferenceError: mesh is not defined
    at completeAgentConnection3 (/opt/meshcentral/node_modules/meshcentral/meshagent.js:752:92)
    at /opt/meshcentral/node_modules/meshcentral/meshagent.js:699:13
    at /opt/meshcentral/node_modules/mongodb/lib/utils.js:410:17
    at executeCallback (/opt/meshcentral/node_modules/mongodb/lib/utils.js:402:9)
    at handleCallback (/opt/meshcentral/node_modules/mongodb/lib/utils.js:128:55)
    at /opt/meshcentral/node_modules/mongodb/lib/operations/cursor_ops.js:224:62
    at handleCallback (/opt/meshcentral/node_modules/mongodb/lib/utils.js:128:55)
    at completeClose (/opt/meshcentral/node_modules/mongodb/lib/cursor.js:898:14)
    at /opt/meshcentral/node_modules/mongodb/lib/cursor.js:909:37
    at Cursor._endSession (/opt/meshcentral/node_modules/mongodb-core/lib/cursor.js:195:5)
```

**Comment by Ylianst** [comment]
Fixed in MeshCentral v0.3.6-d. Please confirm.

**Comment by hunter-nl** [comment]
I do not see this error anymore :-)
